### DISCUSSION TITLE
Can't create guest account using the provided script

### DISCUSSION THREAD

**Discussion by jaum20** [j-fbriere/squawker#229] [Q&A] [answered]
Hi,

I am using the script to create a guest account, but it return nothing. Here is the raw json output after i remove everything after `jq`
[output.json](https://github.com/j-fbriere/squawker/files/14395412/output.json)

what am I doing wrong

**Comment by j-fbriere** [comment, marked as answer]
It is now normal, unfortunately.
Twitter/X do not allow to create guest accounts anymore.
To circumvent the problem you'll have to create an anonym Twitter/X regular account.
The wiki page explains how: [Regular Twitter X accounts](https://github.com/j-fbriere/squawker/wiki/3.-Regular-Twitter-X-accounts)
After that you'll just have to add this account to Squawker (Settings / Account then tap on the Plus icon).

**Comment by jaum20** [comment]
Thank you
